.travis.yml

```diff
@@ -6,13 +6,18 @@ python:
 
 sudo: "required"
 
-before_install:
-  - "pip install virtualenv"
-  - "virtualenv venv"
-  - "source venv/bin/activate"
+env:
+  matrix:
+  - TOXENV=foxpuppet GUI=1
 
-install:
-  - "pip install -r requirements.txt "
+before_install:
+  - pip install mozdownload
+  - mozdownload --type=daily
+  - for f in *; do if [[ $f =~ (mozilla-central|firefox-52) ]]; then tar -xvf $f; fi; break; done
+  - if [ $TOXENV ]; then sudo pip install tox; fi
```

Comment:
You shouldn't need `sudo` for installing Tox. Also, no need for the `if` as we're only using Tox.